src/tracks/ui/TextEditHelper.cpp

```
@@ -96,7 +96,20 @@ bool TextEditHelper::IsSelectionEmpty()
 
 bool TextEditHelper::CaptureKey(int, int mods)
 {
-   return mods == wxMOD_NONE || mods == wxMOD_SHIFT;
+   // Check for modifiers and only allow text-producing modifier combinations
+   if (mods & ~(wxMOD_SHIFT | wxMOD_ALTGR))
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)